crates/ekore/src/anomalous_dimensions/unpolarized/spacelike/as4/gnsm.rs

```diff
@@ -0,0 +1,236 @@
+/// Compute the non-singlet-like non-singlet anomalous dimension.
+use num::complex::Complex;
+use num::traits::Pow;
+
+use crate::cmplx;
+use crate::constants::{CF, ZETA3};
+use crate::harmonics::cache::{Cache, K};
+use crate::harmonics::log_functions::{lm11, lm11m1, lm12m1, lm13m1};
+
+// The routine is taken from [\[Moch:2017uml\]][crate::bib:Moch:2017uml].
+
+// The nf^{0,1} leading large-nc contributions and the nf^2 part are
+// high-accuracy (0.1% or better) parametrizations of the exact
+// results. The nf^3 expression is exact up to numerical truncations.
+
+// The remaining nf^{0,1} terms are approximations based on the first
+// eight odd moments together with small-x and large-x constraints.
+// The two sets spanning the error estimate are called via  IMOD = 1
```

Comment:
I like the literal copy of the comment by Moch et al., but we should replace `IMOD` -> `variation` (or change the name of the argument)

Reply:
done in b0869d11